### PR TITLE
Create oauth for Hubs.md

### PR DESCRIPTION
**Pull request by rowa97**

### CODE REVIEW

File: articles/hubs/oauth.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# OAuth for Hubs 
+
+## What 
+
+OAuth provides a level of security to your documentation to restrict access to it. Enabling OAuth in Hubs can be accomplished through two different methods: via Modeling or directly into the Hubs code.
+
+> Stoplight supports OAuth 1.0 and 2.0
```
Comment (casserni) [inline articles/hubs/oauth.md]: Make sure we support OAuth 1.0 in HUB

File: articles/hubs/oauth.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+1. Create a new Modeling file or select an existing one referenced by the Hub you wish to modify 
+
+> If you are creating a new Modeling file, make sure to reference it in your Hub 
+
+2. Next to **Security** in the left-middle toolbar, Add a Security Scheme by clicking the **+**
+3. Input a **key**
+4. Select oauth2 under **type** 
+5. Select an OAuth **flow** 
+6. Select an **authorization url**
+7. Select a **token url**
```
Comment (casserni) [inline articles/hubs/oauth.md]: 8. Add a **refresh url** (optional)

File: articles/hubs/oauth.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+2. Select **Code View**
+3. Add the following lines of code to your Hub:
+
+```
+"config": {
+    "http": {
+      "oauth2": {
+        "credentials": {
+          "authorize_url": "insert authorize url here",
+          "access_token_url": "insert access token url here",
```
Comment (casserni) [inline articles/hubs/oauth.md]: close your curly braces both here and in the photo

File: articles/hubs/oauth.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+OAuth provides a level of security to your documentation to restrict access to it. Enabling OAuth in Hubs can be accomplished through two different methods: via Modeling or directly into the Hubs code.
+
+> Stoplight supports OAuth 1.0 and 2.0 
+
+## How 
+
+### Modeling 
+
+![OAuth in Modeling](https://github.com/stoplightio/docs/blob/develop/assets/images/hubs-oauth-modeling.png?raw=true)
```
Comment (casserni) [inline articles/hubs/oauth.md]: might need a new picture, the ui is going to have an extra input

File: articles/hubs/oauth.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+1. Create a new Modeling file or select an existing one referenced by the Hub you wish to modify 
+
+> If you are creating a new Modeling file, make sure to reference it in your Hub 
+
+2. Next to **Security** in the left-middle toolbar, Add a Security Scheme by clicking the **+**
+3. Input a **key**
+4. Select oauth2 under **type** 
+5. Select an OAuth **flow**
```
Comment (casserni) [inline articles/hubs/oauth.md]: depending on the flow there will be different inputs

Comment (casserni) [reply articles/hubs/oauth.md]: Although I think we only support the accessCode flow right now in hubs? @marbemac 
so we might want to clarify to specifically choose that flow

File: articles/hubs/oauth.md (hunk 1 above)
Comment (rowa97) [reply articles/hubs/oauth.md]: we do, it's weird though because OpenAPI doesn't, so it can't be included in modeling

**Comment by rowa97** [comment]
made the changes

**Comment by rowa97** [comment]
needs an updated modeling security image when UI is updated

**Pull request merged**